skills/ios-simulator-testing/scripts/app_state_capture.py

```
@@ -0,0 +1,292 @@
+        """Capture screenshot of current screen."""
+        cmd = ["xcrun", "simctl", "io"]
+
+        if self.udid:
+            cmd.append(self.udid)
+        else:
+            cmd.append("booted")
+
+        cmd.extend(["screenshot", str(output_path)])
+
+        try:
+            subprocess.run(cmd, capture_output=True, check=True)
```

Ruff (0.14.1)

`subprocess` call: check for execution of untrusted input

(S603)